packages/common/src/index.ts

```diff
@@ -683,11 +694,7 @@ export type HealthState = {
         writeKey: string;
         dataPlaneUrl: string;
     };
-    sentry: {
-        dsn: string;
-        environment: string;
-        release: string;
-    };
```

Comment:
Not sure why we need to expose the backend dsn

Reply:
done: https://github.com/lightdash/lightdash/pull/9910/commits/b85e2e7cdb73248ae1f6446bbc35442c34911156